### PR TITLE
feat(assistant V2): add support for agent description

### PR DESCRIPTION
**Pull request by fontanierh**
Field nullable in DB, but required by API and frontend.

### CODE REVIEW

File: front/types/assistant/agent.ts [hunk 1]
```diff
@@ -67,6 +67,7 @@ export type AgentConfigurationType = {
   status: AgentConfigurationStatus;
 
   name: string;
+  description: string | null;
```
Comment (PopDaph) [inline front/types/assistant/agent.ts:70]: Why allowing null?

**Review by PopDaph — DISMISSED** [review]
Approving  already bcause code looks good, I would only remove the nullable option but I let you decide 🙏🏻

File: front/types/assistant/agent.ts (hunk 1 above)
Comment (spolu) [reply front/types/assistant/agent.ts:70]: Let's not allow null

File: front/pages/w/[wId]/builder/assistants/index.tsx [hunk 2]
```diff
@@ -99,7 +99,9 @@ export default function AssistantsBuilder({
               key={agent.sId}
               imageUrl={agent.pictureUrl}
               name={`@${agent.name}`}
-              description={"lorem ipsum dolor sit amet"}
+              description={
+                agent.description || `The "${agent.name}" assistant.`
```
Comment (spolu) [inline front/pages/w/[wId]/builder/assistants/index.tsx:103]: so we remove this

**Review by spolu — CHANGES_REQUESTED** [review]

File: front/lib/api/assistant/global_agents.ts [hunk 3]
```diff
@@ -90,6 +92,7 @@ async function _getSlackGlobalAgent(
     id: GLOBAL_AGENTS_ID.Slack,
     sId: GLOBAL_AGENTS_SID.Slack,
     name: "slack",
+    description: "Your Slack workspace.",
```
Comment (spolu) [inline front/lib/api/assistant/global_agents.ts:95]: An assistant with context on your Slack workspace

**Review by spolu — CHANGES_REQUESTED** [review]

File: front/types/assistant/agent.ts (hunk 1 above)
Comment (fontanierh) [reply front/types/assistant/agent.ts:70]: For existing objects ?

Comment (spolu) [reply front/types/assistant/agent.ts:70]: DELETE them

**Comment by fontanierh** [comment]
fixed

**Pull request merged**